internal/artieclient/destination.go

```diff
@@ -47,14 +50,14 @@ func (dc DestinationClient) Get(ctx context.Context, destinationUUID string) (De
 	return makeRequest[Destination](ctx, dc.client, http.MethodGet, path, nil)
 }
 
-func (dc DestinationClient) Create(ctx context.Context, type_, label string, config DestinationSharedConfig, sshTunnelUUID *uuid.UUID) (Destination, error) {
+func (dc DestinationClient) Create(ctx context.Context, destination BaseDestination) (Destination, error) {
 	body := map[string]any{
-		"type":         type_,
-		"label":        label,
-		"sharedConfig": config,
+		"type":         destination.Type,
+		"label":        destination.Label,
+		"sharedConfig": destination.Config,
 	}
-	if sshTunnelUUID != nil {
-		body["sshTunnelUUID"] = *sshTunnelUUID
+	if destination.SSHTunnelUUID != nil {
+		body["sshTunnelUUID"] = *destination.SSHTunnelUUID
```

Comment:
Could you add:
`"sshTunnelUUID": destination.SSHTunnelUUID` to the `body` initialization?